src/funtracks/import_export/export_to_geff.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import (
+    TYPE_CHECKING,
+)
+
+import geff
+import networkx as nx
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from funtracks.data_model.tracks import Tracks
+
+
+def export_to_geff(tracks: Tracks, directory: Path, overwrite: bool = False):
+    """Export the Tracks nxgraph to geff.
+
+    Args:
+        tracks (Tracks): Tracks object containing a graph to save.
+        directory (Path): Destination directory for saving the Zarr.
+        overwrite (bool): If True, allows writing into a non-empty directory.
+
+    Raises:
+        ValueError: If the path is invalid, parent doesn't exist, is not a directory,
+                    or if the directory is not empty and overwrite is False.
+    """
+    directory = directory.resolve(strict=False)
+
+    # Ensure parent directory exists
+    parent = directory.parent
+    if not parent.exists():
+        raise ValueError(f"Parent directory {parent} does not exist.")
+
+    # Check target directory
+    if directory.exists():
+        if not directory.is_dir():
+            raise ValueError(f"Provided path {directory} exists but is not a directory.")
+        if any(directory.iterdir()) and not overwrite:
+            raise ValueError(
+                f"Directory {directory} is not empty. Use overwrite=True to allow export."
+            )
+    else:
+        # Create dir
+        directory.mkdir()
+
+    # update the graph to split the position into separate attrs, if they are currently
+    # together in a list
+    if isinstance(tracks.pos_attr, str):
+        graph = split_position_attr(tracks)
+    else:
+        graph = tracks.graph
+
+    axis_names = (
```

Comment:
If the pos_attr wasn't a single string, but already a tuple, we should probably use those axis names.

Reply:
Yes makes sense, I fixed it!